hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -550,6 +631,17 @@ public AbfsRestOperation checkAccess(String path, String rwx)
     return op;
   }
 
+  private void appendSASTokenToQuery(String path, String operation, AbfsUriQueryBuilder queryBuilder) throws SASTokenProviderException {
+    try {
+      LOG.trace("Fetch SAS token for {}", path);
```

Comment:
add operation to these traces; it'll be handy in production

Reply:
Added